### DISCUSSION TITLE
v2.3.8970

### DISCUSSION THREAD

**Discussion by gbakeman** [nutdotnet/WinNUT-Client#169] [Announcements]
Sorry about the broken installer if you were able to download it in the last pre-release. That led me to deploy an update in AGauge which not only fixed that, but another older bug as well. I guess I can't blame this one on CrowdStrike. 😅

## What's Changed
* Separate AGauge into a NuGet package, with inherited control in https://github.com/nutdotnet/WinNUT-Client/pull/59
* locale: add Ukrainian language by @Vladdrako in https://github.com/nutdotnet/WinNUT-Client/pull/160
* More efficient connection handling, better suspend/resume support in https://github.com/nutdotnet/WinNUT-Client/pull/163
  * Better tolerance of NUT servers behaving strangely (s/o to Synology)
  * Login command now works correctly, and is not critical for connecting to your UPS. Try deleting your username and password and see if everything still works.
  * Official support for network protocol version 1.3
  * Polling interval is no longer ignored (always defaulted to 3.0 seconds)
* Revert Polling Interval display to seconds in https://github.com/nutdotnet/WinNUT-Client/pull/164
* AGauge - fix tick mark bug in https://github.com/nutdotnet/WinNUT-Client/pull/168
  * Also fixes the broken installer in #167.

**Hotfix**: The Ukrainian translation was not building - that has been corrected (and MSI/zip have been updated too.)

## New Contributors
* @Vladdrako made their first contribution in https://github.com/nutdotnet/WinNUT-Client/pull/160

**Full Changelog**: https://github.com/nutdotnet/WinNUT-Client/compare/v2.3.8892...v2.3.8970

<hr /><em>This discussion was created from the release <a href='https://github.com/nutdotnet/WinNUT-Client/releases/tag/v2.3.8970'>v2.3.8970</a>.</em>

**Comment by masterwishx** [comment]
Seems working fine after updated from app

**Comment by Vladdrako** [comment]
Ukrainian language is missing? :)
![image](https://github.com/user-attachments/assets/9bf4089b-49fa-4cc4-9f4d-d39c1633af82)

**Reply by gbakeman** [reply]
Sorry about that - I'm not very experienced adding new languages to WinNUT. Problem is fixed now, I think. Please redownload and let me know if it's working for you. And thank you for taking the time to add this translation, I hope you'll continue to add more in the future!

**Reply by Vladdrako** [reply]
> Sorry about that - I'm not very experienced adding new languages to WinNUT. Problem is fixed now, I think. Please redownload and let me know if it's working for you. And thank you for taking the time to add this translation, I hope you'll continue to add more in the future!

Now it's here, many thanks :)
